packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -133,18 +147,18 @@ class MouseTracker extends ChangeNotifier {
     _trackedAnnotations.remove(annotation);
   }
 
-  bool _postFrameCheckScheduled = false;
-  void _scheduleMousePositionCheck() {
+  final Set<int> _postFrameCheckDeviceIds = <int>{};
```

Comment:
use a HashSet unless you care about order

Reply:
Actually, now that I'm only doing this for all the device IDs, I don't think I even need the set: a bool will do.